### ISSUE TITLE
NTR, pulmo: Paradoxical embolism

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6480]
**Preferred term label:**
Paradoxical embolism

**Synonyms**
Paradoxic embolism

**Definition (free text, please give PubMed ID)**
Paradoxical embolism describes the passage of a venous or right-sided cardiac thrombus into the arterial or systemic circulation. 
Comment: Paradoxical embolism occurs most commonly through an intracardiac defect at the atrial level, but can also be due to another type of right-to-left-shunt, as for example in pulmonary arteriovenous malformations. Paradoxical embolism may lead to serious complications as ischemic stroke.
•	PMID: 11700487
•	PMID: 12130431 

**Parent term (use hpo.jax.org/app)**
Thromboembolism HP:0001907
Pulmonary embolism HP:0002204

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
I am wondering if it would be better to use *just* Thromboembolism HP:0001907 as the parent term?

**Comment by pnrobinson** [comment]
added
